Src/MoneyFox.Core/ApplicationCore/Queries/Statistics/GetCategorySummary/GetCategeorySummaryQuery.cs

```diff
@@ -143,7 +143,7 @@ private static decimal SumForCategory(IEnumerable<Payment> payments)
                 return sumForCategory;
             }
 
-            return Math.Round(d: sumForCategory / (timeDiff.Days / DAY_DIVIDER), decimals: POSITIONS_TO_ROUND, mode: MidpointRounding.ToEven);
+            return Math.Round(d: (decimal)(sumForCategory) / (timeDiff.Days / DAY_DIVIDER), decimals: POSITIONS_TO_ROUND, mode: MidpointRounding.ToEven);
```

Comment:
I think this decimal is for free since `sumForCategory` is a decimal. VS / Rider would also remove that with the next code cleanup due to that reason. Or is there a reason why you explicitly stated this?

Reply:
@NPadrutt, I think simplicity and pure code is more important to u, I will cancel change, commit will be under your authorship.

Reply:
@NPadrutt, fixed here: https://github.com/MoneyFox/MoneyFox/pull/2344/commits/ae20d5b78031a156a05799632d8b64e7f6a002a5